src/Runner/TestHandler.php

```diff
@@ -146,6 +146,10 @@ private function initiateDataProvider(Test $test, string $provider)
 		try {
 			[$dataFile, $query, $optional] = Tester\DataProvider::parseAnnotation($provider, $test->getFile());
 			$data = Tester\DataProvider::load($dataFile, $query);
+			if (count($data) < 1) {
```

Comment:
A co tady dát `(!$optional && !count($data))`, tedy umožnit volitelně i prázdná data?

Reply:
Pokud je optional, bude SKIPPED a to je myslím v pohodě. Jinak by ten test opět nebyl vůbec v seznamu jako `.` ani `s`.

Reply:
No jasně, výjimka se hned zachytí. 🤦‍♂️